core/src/main/scala/cats/eq/Is.scala

```diff
@@ -0,0 +1,64 @@
+package cats.eq
+
+import cats.Id
+
+/**
+ * A value of `A Is B` witnesses that the types `A` and `B` are the same.
+ * You can use this value to coerce `A` into `B` and therefore `A Is B` is
+ * similar in feel to `A =:= B`; however, `A Is B` has more computational
```

Comment:
I'm not precisely clear how to add that to the documentation appropriately. Guidance very welcome!